parking_lot/parking_lot.py

```diff
@@ -1,8 +1,150 @@
 # -*- coding: utf-8 -*-
-from flask import Flask
+import datetime
 
-app = Flask(__name__)
 
-@app.route("/")
-def hello_world():
-    return "<p>Hello, World!</p>"
+class Parking_lot_error( Exception ):
+    message = 'unhandle error with parking lot'
```

Comment:
the purpose of having a class attribute is to share it with others classes and instances, but in this case every class is overriding it, do we really need this?

Reply:
the message is for the rest api and is changing in each because change the message of the exception but not other parts.